resolwe_bio/docker_images/dnaseq/assets/amplicon_html_template.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+
+        <!-- Necessary bokeh includes: -->
+        <link href="assets.12.13.min.css" type="text/css"  rel="stylesheet" />
```

Comment:
Let's see what happens [here](https://github.com/genialis/resolwe-bio/pull/540) first and merge this PR afterwards.